presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergOrcFileWriter.java

```diff
@@ -31,6 +31,7 @@
 import io.prestosql.orc.metadata.statistics.DoubleStatistics;
 import io.prestosql.orc.metadata.statistics.IntegerStatistics;
 import io.prestosql.orc.metadata.statistics.StringStatistics;
+import io.prestosql.plugin.hive.WriterKind;
```

Comment:
As the WriterKind is being used across plugins/connectors, should we move it out of the hive plugin module?

Reply:
`WriterKind` is specific to presto-hive's `OrcFileWriter`, which is used by both presto-hive and presto-iceberg.  Module presto-iceberg depends on presto-hive.  I don't think it makes sense to move WriterKind into presto-spi, because not all connectors will need the distinction between delete writers and insert writers.  We need to for `OrcFileWriter` because the files written for `DELETE` and `INSERT` have different formats in `ORC ACID`.

Reply:
Sure, we can have it here. With other connectors if we find a similar need, we can move it later